pkgs/blast_repo/lib/src/tweaks/no_reponse_tweak.dart

```diff
@@ -20,75 +16,59 @@ class NoResponseTweak extends ExactFileTweak {
           id: 'no-response',
           description:
               "configure a 'no response' bot to handle needs-info labels",
-          filePath: _filePath,
+          filePath: filePath,
         );
 
   @override
   bool get stable => false;
 
   @override
-  String expectedContent(Directory checkout, String repoSlug) =>
-      _noResponseContent;
-
-  @override
-  FutureOr<FixResult> fix(Directory checkout, String repoSlug) {
-    // Check for and fail if this fix is not being run for a dart-lang repo.
-    if (!isDartLangOrgRepo(checkout)) {
-      print('  repo is not in the dart-lang/ org');
-      return FixResult.noFixesMade;
-    }
+  String expectedContent(Directory checkout, String repoSlug) {
+    final org = repoSlug.split('/').first;
 
-    return super.fix(checkout, repoSlug);
+    // Substitute for the github org value.
+    return noResponseContent.replaceAll('{org}', org);
   }
 }
 
-bool isDartLangOrgRepo(Directory checkout) {
-  final gitConfigFile = File(path.join(checkout.path, '.git', 'config'));
-  final contents = gitConfigFile.readAsStringSync();
-
-  return contents.contains('@github.com:dart-lang/') ||
-      contents.contains('github.com/dart-lang/');
-}
-
-const _filePath = '.github/workflows/no-response.yml';
-
-// The content below - as is - should only be run on dart-lang repos.
-// Alternatively, we could substitue in the GitHub org for the repo into the
-// config content.
+const filePath = '.github/workflows/no-response.yml';
```

Comment:
does this need to be public?

Reply:
No, but this whole lib is src/ - so not API - and I thought it read a little cleaner w/o the underscore.

Reply:
Expose what needs exposing. Then when things aren't used they can be easily found...and removed.

And it's easier to reason about the "exposed" surface area of a library.